christopherjohnson1/daily-journal
# Cj record mood from json

# Steps to test

1. `git fetch --all`
1. `git checkout cj-record-mood-from-json`
1. make sure you are able to save an entry , and the PK and FK are used
